static/styles/informational_overlays.css

```diff
@@ -109,7 +109,7 @@
 
     td:not(.definition) {
         text-align: left;
-        white-space: nowrap;
+        white-space: normal;
```

Comment:
Can you post before/after screenshots of this change at various sizes to confirm that it works as intended?

Reply:
I have added some more screenshots [here](https://github.com/zulip/zulip/pull/16823) with different widths and heights where it was overflowing.

Reply:
Great!  I think there's some sort of line-height issue that makes the keyboard shortcuts wrapping not look great in a few cases; can you try tweaking that?

Reply:
I have added the spacing between each ```<kbd>``` element.  
I have also updated the screenshots. 
Can you check It now?